ccm_web/client/src/components/Home.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+
+import FeatureCard, { FeatureCardProps } from './FeatureCard'
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined'
+import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined'
+import MergeTypeIcon from '@material-ui/icons/MergeType'
+import PersonAddIcon from '@material-ui/icons/PersonAdd'
+import PersonAddOutlinedIcon from '@material-ui/icons/PersonAddOutlined'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    minWidth: 275
+  }
+}))
+
+function Home (): JSX.Element {
+  const classes = useStyles()
+
+  const mergeSectionProps: FeatureCardProps = {
+    title: 'Merge Sections',
+    description: 'Combine sections taught by yourself or others into one Canvas site for easier management',
+    icon: <MergeTypeIcon fontSize='large'/>,
+    size: 'tall'
+  }
+
+  const gradebookToolsProps: FeatureCardProps = {
+    title: 'Gradebook Tools',
+    description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+    icon: <LibraryBooksOutlinedIcon fontSize='large'/>,
+    size: 'short'
+  }
+
+  const addSectionsGroupsProps: FeatureCardProps = {
+    title: 'Add Sections / Groups',
+    description: 'Add sections, groups through csv viles into your own course',
+    icon: <AccountCircleOutlinedIcon fontSize='large'/>,
+    size: 'tall'
+  }
+
+  const addUMUsersProps: FeatureCardProps = {
+    title: 'Add UM Users',
+    description: 'Add UM users to your available sections',
+    icon: <PersonAddIcon fontSize='small'/>,
+    size: 'short'
+  }
+
+  const addNonUMUsersProps: FeatureCardProps = {
+    title: 'Add Non-UM Users',
+    description: 'Enroll non-UM users to your available sections',
+    icon: <PersonAddOutlinedIcon fontSize='small'/>,
+    size: 'short'
+  }
```

Comment:
Just a thought, can this entire section can be separated to a separate component and can be made as list

Reply:
I think we wait to see how this is going to be integrated with the backend/role information for now.

There is little to the Home component besides this information as it is.

Reply:
I'm fine waiting as well, this can be refactored later.